### ISSUE TITLE
Add support to query the amount of frames in a trace

### ISSUE THREAD

**Issue by tanty** [LunarG/gfxreconstruct#329]
Currently, the only way I see to know the amount of frames in a trace is by reproducing the whole trace and parsing the STDOUT. For example:

```
$ gfxrecon-replay vkcube.gfxr
46.201665 fps, 0.216442 seconds, 10 frames, 1 loop, framerange 1-10
```

It would be useful to be able to query for this (and other info) about the trace without having to actually replay it.

Apitrace has the "dump" command which allows this with, [for example](https://github.com/apitrace/apitrace/blob/master/docs/USAGE.markdown#call-sets):
```
$ apitrace dump --calls=frame foo.trace
```

**Comment by dustin-lunarg** [comment]
I created a simple command line utility to print information about capture files, including the frame count.  The change can be found in PR #333.  The output currently looks something like this:
```
D:\>gfxrecon-info.exe D:\temp\test_20200224T181507.gfxr
Total frames: 138

Application info:
        Application name: vkcube
        Application version: 0
        Engine name: vkcube
        Engine version: 0
        Target API version: 4194304

Physical device info:
        Device name: Radeon RX Vega
        Device ID: 0x687f
        Vendor ID: 0x1002
        Driver version: 8388735 (0x80007f)
        API version: 4202627

Device memory allocation info:
        Total allocations: 5
        Min allocation size: 1216
        Max allocation size: 590172

Pipeline info:
        Total graphics pipelines: 1
        Total compute pipelines: 0
```

Is this the type of functionality you are asking for?

If you are instead interested in a utility that writes function names and parameters to the console or to a file, there is currently some work in progress for this feature.  We are tracking it with issue #158.

**Comment by tanty** [comment]
> Is this the type of functionality you are asking for?

Yes. Thanks @dustin-lunarg, this is already enough for this enhancement request.

**Comment by dustin-lunarg** [comment]
Merged #333 to the dev branch.
